server/src/main/kotlin/org/example/myhome/server/device/DeviceServer.kt

```diff
@@ -23,16 +23,20 @@ open class DeviceServer(
   private val workerGroup: NioEventLoopGroup,
   private val deviceRegisterService: DeviceRegisterService
 ) {
+
+//  sharable channel handlers
+  private val lengthFieldPrepender = LengthFieldPrepender(4)
+  private val simpCodec = SimpCodec()
+  private val loggingHandler = LoggingHandler()
+
   open fun start(): ChannelFuture? {
     val init = object : ChannelInitializer<SocketChannel>() {
       override fun initChannel(ch: SocketChannel?) {
+        ch?.pipeline()?.addFirst("lengthDecoder", LengthFieldBasedFrameDecoder(256, 0, 4, 0, 4))
```

Comment:
please extract to the constants this magic numbers, and reuse them!

Reply:
What do you think if I extract shared constants to some file like DeviceServerConstants.kt?